ashpd-demo/Cargo.toml

```diff
@@ -7,7 +7,7 @@ version = "0.4.1"
 [dependencies]
 adw = {version = "0.6", package = "libadwaita", features = ["v1_4"]}
 anyhow = "1.0"
-ashpd = {version = "^0.8", features = ["gtk4", "tracing", "pipewire"]}
+ashpd = {path="..", features = ["gtk4", "tracing", "pipewire"]}
```

Comment:
This will not work for flatpak, you will have to point it to your github branch until the PR is approved and then merged.

Reply:
ugh.